File: QuestoesBeecrowd-Iniciante/1040.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>

int main()
{
    setlocale(LC_ALL, "pt-br");
    float n1, n2, n3, n4, nexame;

    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);

    float media = (n1 * 2 + n2 * 3 + n3 * 4 + n4) / 10;
    printf("MEDIA: %.1f\n", media);
    if (media >= 7)
    {
        printf("Aluno aprovado.\n");
    }
    else if (media < 5)
    {
        printf("Aluno reprovado.\n");
    }

    else
    {
        printf("Aluno em exame.\n");
        scanf("%f", &nexame);
        printf("Nota do exame: %.1f\n", nexame);
        nexame = (media + nexame) / 2;
        if (nexame >= 5)
        {
            printf("Aluno aprovado.\n");
            printf("Media final: %.1f\n", nexame);
        }
        else
        {
            printf("Aluno reprovado.\n");
        }
    }

    return 0;
}

File: src/Iniciante/1040.java
package Iniciante;

import java.util.Scanner;

public class a1040 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        float a = input.nextFloat();
        float b = input.nextFloat();
        float c = input.nextFloat();
        float d = input.nextFloat();
        float media = (a * 2 + b * 3 + c * 4 + d) / 10;
        float exame = 0, media2;
        String mm = String.format("%.1f", media);
        System.out.println("Media: " + mm);
        if (media >= 5 && media <= 6.9) {
            exame = input.nextFloat();

        }
        if (media >= 7) {

            System.out.println("Aluno aprovado.");
        } else if (media < 5) {
            System.out.println("Aluno reprovado.");
        } else if (media >= 5 && media <= 6.9) {
            System.out.println("Aluno em exame.");
            media2 = (media + exame) / 2;
            System.out.println("Nota o exame:" + exame);

            if (media2 >= 5) {
                System.out.println("Aluno aprovado.");
                String me2 = String.format("%.1f", media2);
                System.out.println("Media final: " + me2);
            } else if (media2 < 5) {
                System.out.println("Aluno reprovado.");
                String me3 = String.format("%.1f", media2);
                System.out.println("Media final: " + me3);
            }

        }

        input.close();
    }
}
